### DISCUSSION TITLE
Weird training and test losses and accuracies in Module 4

### DISCUSSION THREAD

**Discussion by a-d14** [mrdbourke/pytorch-deep-learning#296] [Q&A]
Guys, when I plot the training and test accuracies with and without data augmentation these are the graphs I get...
<img width="632" alt="Screen Shot 2023-02-10 at 9 47 59 AM" src="https://user-images.githubusercontent.com/26755675/218134887-205cf5e2-cb55-4b2e-af3a-228050a18f16.png">
As you can see, the test loss for model 0 is higher than model 1 but test accuracy for model 0 is also higher than model 1. How is that possible?

**Comment by nazarPuriy** [comment]
From my point of view you are training the model for few epochs so the behaviour is kind of erratic when you start with fresh model.

**Comment by mrdbourke** [comment]
I'm with @nazarPuriy on this.

Perhaps trying to train the model for longer might result in better curves? 

If the model is capable of learning something, the loss should trend downwards over time.

Is there anything else you've tried?
